cli/src/main/java/com/box/l10n/mojito/cli/command/PushCommand.java

```diff
@@ -146,4 +164,28 @@ public void execute() throws CommandException {
 
         consoleWriter.fg(Ansi.Color.GREEN).newLine().a("Finished").println(2);
     }
+
+    /**
+     * Get fileMatch list contains git changes against git tree HEAD
+     * @param sourceFileMatches
+     * @return
+     * @throws CommandException
+     */
+    private ArrayList<FileMatch> getGitChangeFiles(ArrayList<FileMatch> sourceFileMatches) throws CommandException {
+        GitRepository gitRepository = new GitRepository();
+        gitRepository.init(commandDirectories.getSourceDirectoryPath().toString());
+        Git git = new Git(gitRepository.jgitRepository);
+        String gitPath = gitRepository.getDirectory().getPath().replace(".git", "");
+        Set<String> gitDiffFiles = new HashSet<>();
```

Comment:
can we have logic to skip some diff, the equivalent of ``` if [ `git diff | grep "^\+" | grep -c -v -E "([+]{3})|(^\+\"POT-Creation-Date:)"` != '0' ]; then``` for the POT files

Reply:
going to add